README.md

```diff
@@ -54,7 +54,7 @@ A quick demo of how to submit a calculation using the provided example files:
 verdi daemon start     # make sure the daemon is running
 cd examples/calculations
 verdi run submit_singlepoint.py "janus@localhost" --architecture mace --model "/path/to/model"    # run singlepoint calculation
-verdi run submit_geomopt.py "janus@localhost" --structure "path/to/structure" --model "path/to/model" --steps 5 --fully-opt True # run geometry optimisation
+verdi run submit_geomopt.py "janus@localhost" --structure "path/to/structure" --model "path/to/model" --steps 5 --fully_opt True # run geometry optimisation
```

Comment:
A similar note to before: using underscores differs from the default for click/typer option naming (see `Parameters` [here](https://click.palletsprojects.com/en/8.1.x/api/)) (and so also from janus currently). Are you sure we want this?

Reply:
I know, this was something I was pointing out when I asked you about the underscore. basically if I put fully-opt rather than fully_opt I get an error that it does not like it, maybe because of the way aiida itself is written (?), so I decided to make it with the _ also in click. Still can change it tho

Reply:
[Random example](https://aiida-quantumespresso.readthedocs.io/en/latest/reference/cli/index.html#cmdoption-aiida-quantumespresso-calculation-launch-cp-F), but looks like it should work ok. We can have a look tomorrow maybe